### PR TITLE
Instrument log with taker_id

### PR DESCRIPTION
**Pull request by bonomat**
This allows for easier log analysis

Would be nice to backport this to 0.4.x so that we get better logs for the maker

### CODE REVIEW

File: daemon/src/maker_cfd.rs [hunk 1]
```diff
@@ -194,7 +194,7 @@ where
         taker_id: Identity,
         ctx: &mut xtra::Context<Self>,
     ) -> Result<()> {
-        tracing::info!(%order_id, "Received proposal from taker {taker_id}");
+        tracing::info!(%order_id, %taker_id,  "Received rollover proposal from taker");
```
Comment (thomaseizinger) [inline daemon/src/maker_cfd.rs:197]: I think this one is duplicated with the protocol actor.

**Review by thomaseizinger — APPROVED** [review]

File: daemon/src/maker_cfd.rs (hunk 1 above)
Comment (bonomat) [reply daemon/src/maker_cfd.rs:197]: Can't find it.

Comment (thomaseizinger) [reply daemon/src/maker_cfd.rs:197]: https://github.com/itchysats/itchysats/blob/6e4d2da18c2bd29528fd9379223880d66a153519/daemon/src/rollover_maker.rs#L236-L239

**Pull request merged**